### DISCUSSION TITLE
How long does it usually take for updated Sopel to show up in pip?

### DISCUSSION THREAD

**Discussion by antdude** [sopel-irc/sopel#2155] [Q&A] [answered]
I don't see v7.1.2 from pip after seeing https://github.com/sopel-irc/sopel/releases/tag/v7.1.2.

Thank you for reading and hopefully answering soon. :)

**Comment by dgw** [comment, marked as answer]
Publishing the GitHub release is what triggers the process of uploading a new PyPI release to _start_ (it's automated), so it takes a few minutes. Bit longer this time because I tagged the wrong branch at first, meaning the process had to start over.

**Comment by antdude** [comment]
Thanks dgw. I will check later in pip (pip install --upgrade sopel).
